### PR TITLE
60 base sensors

### PR DESCRIPTION
**Pull request by David-Rey**

### CODE REVIEW

File: EosPayload/drivers/GPS_driver.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def setup(self) -> None:
 
     @staticmethod
     def enabled() -> bool:
-        return True
+        return False
```
Comment (DarylDohner) [inline EosPayload/drivers/GPS_driver.py:46]: re-enable

File: EosPayload/drivers/cutdown_driver.py [hunk 2]
```diff
@@ -10,15 +10,16 @@
...
 
     @staticmethod
     def enabled() -> bool:
-        return False
+        return True
 
     @staticmethod
     def get_device_id() -> Device:
-        return Device.MISC_ENGINEERING_2
+        return Device.MISC_ENGINEERING_3
```
Comment (DarylDohner) [inline EosPayload/drivers/cutdown_driver.py:22]: Device.CUTDOWN

File: EosPayload/drivers/cutdown_driver.py [hunk 3]
```diff
@@ -42,12 +43,15 @@ def setup(self):
...
+                if altitude > CutdownDriver.auto_cutdown_altitude and not self.has_triggered:
+                    self.has_triggered = True
+                    self.cutdown_trigger()
+            except TypeError:
+                logger.info("No Altitude Data")
+
+            time.sleep(5)
 
     def cutdown_trigger(self):
         self._logger.info("~~~PULLING PIN HIGH~~~")
```
Comment (DarylDohner) [inline EosPayload/drivers/cutdown_driver.py:57]: cutdown_trigger_mqtt is not thread safe, you need to use a queue

File: EosPayload/drivers/mock_descent_driver.py [hunk 4]
```diff
@@ -11,11 +12,11 @@ class MockDescentDriver(EngineeringDataDriver):
     """
     def __init__(self, output_directory: str):
         super().__init__(output_directory)
-        self.current_altitude = 35000.0
+        self.current_altitude = 20000.0
 
     @staticmethod
     def enabled() -> bool:
-        return False
+        return True
```
Comment (DarylDohner) [inline EosPayload/drivers/mock_descent_driver.py:19]: disable

File: EosPayload/drivers/science_driver.py [hunk 5]
```diff
@@ -25,11 +25,11 @@ def get_device_name() -> str:
 
     @staticmethod
     def read_thread_enabled() -> bool:
-        return True
+        return False
```
Comment (DarylDohner) [inline EosPayload/drivers/science_driver.py:28]: re-enable

File: EosPayload/drivers/science_driver.py [hunk 6]
```diff
@@ -25,11 +25,11 @@ def get_device_name() -> str:
...
 
     def device_read(self, logger: logging.Logger) -> None:
         reset_pin = None
-        i2c = busio.I2C(board.SCL, board.SDA)
+        i2c = busio.I2C(board.SCL, board.SDA) # this needs to be changed to use I2C1
```
Comment (DarylDohner) [inline EosPayload/drivers/science_driver.py:32]: merge main to get the latest changes

File: requirements.txt [hunk 7]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/VIP-LES/EosLib@v2.0.0#egg=EosLib
+git+https://github.com/VIP-LES/EosLib@55-eoslib-bump#egg=EosLib
```
Comment (DarylDohner) [inline requirements.txt:1]: v3.0.0

**Review by DarylDohner — CHANGES_REQUESTED** [review]

File: config.json [hunk 8]
```diff
@@ -1,34 +1,54 @@
 {
-  "payload_name": "Kian_2",
+  "payload_name": "Gabe_1",
```
Comment (thomasmholder) [inline config.json:2]: Does it need to be Gabe_1 if we're only flying one bone?

File: EosPayload/drivers/telemetry_I2C_driver.py [hunk 9]
```diff
@@ -9,7 +10,7 @@
 
 from EosPayload.lib.base_drivers.driver_base import DriverBase
 from EosLib.format.telemetry_data import TelemetryData
-from adafruit_blinka.microcontroller.am335x import pin
+#from adafruit_blinka.microcontroller.am335x import pin
```
Comment (thomasmholder) [inline EosPayload/drivers/telemetry_I2C_driver.py:13]: What's the motivation to comment this out vs deleting?

File: EosPayload/drivers/telemetry_I2C_driver.py [hunk 10]
```diff
@@ -26,7 +27,8 @@ def setup(self) -> None:
 
     def device_read(self, logger: logging.Logger) -> None:
         logger.info("Starting to poll for data!")
-        self.i2c = busio.I2C(pin.I2C1_SCL, pin.I2C1_SDA)
+        #self.i2c = busio.I2C(pin.I2C1_SCL, pin.I2C1_SDA)
```
Comment (thomasmholder) [inline EosPayload/drivers/telemetry_I2C_driver.py:30]: What's the motivation to comment this out vs deleting?

File: EosPayload/archived_drivers/engineering_data_driver.py [hunk 11]
```diff
@@ -6,6 +6,7 @@
 
 from EosLib.packet.definitions import Type, Priority
 from EosLib.device import Device
+
```
Comment (thomasmholder) [inline EosPayload/archived_drivers/engineering_data_driver.py:9]: Lol

File: EosPayload/drivers/RadioDriver.py [hunk 12]
```diff
@@ -112,6 +115,7 @@ def device_command(self, logger: logging.Logger) -> None:
             (priority, timestamp, packet) = self._thread_queue.get()
             logger.info(f":: = {packet.body}")
             try:
+                # logger.info(packet)
```
Comment (thomasmholder) [inline EosPayload/drivers/RadioDriver.py:118]: Why?

File: EosPayload/drivers/cutdown_driver.py [hunk 13]
```diff
@@ -0,0 +1,97 @@
+from queue import Queue
```
Comment (thomasmholder) [inline EosPayload/drivers/cutdown_driver.py:1]: This driver needs updating to be compatible with the new driver handling, not sure if that's a problem for another issue

**Review by thomasmholder — CHANGES_REQUESTED** [review]
Looks good, wasn't sure what is/isn't in scope

File: config.json (hunk 8 above)
Comment (DarylDohner) [reply config.json:2]: You never explained what the number was for so I just assumed it was just an integer that gets incremented arbitrarily lol

**Review by thomasmholder — DISMISSED** [review]
Looks "good" to me

**Review by thomasmholder — DISMISSED** [review]
Looks good

**Review by thomasmholder — APPROVED** [review]
Aside from the everything this looks good

**Review by DarylDohner — APPROVED** [review]
i'm daryl dohner and i approve these changes

**Pull request merged**